Views/all_tasks.phtml

```diff
@@ -0,0 +1,70 @@
+<form action="/task/deleteTask" method="post">
+    <table border="1">
+        <tr>
+            <td>id</td>
+            <td>Время создания</td>
+            <td>Создал</td>
+            <td>Исполнитель</td>
+            <td>Задание</td>
+            <td>статус</td>
+            <td>Время начала</td>
+            <td>Время окончания</td>
+            <td>Комментарий</td>
+            <td>Подтверждено кем</td>
+            <td>Изображение</td>
+            <td>
+                <button type="submit">Удалить</button>
+            </td>
+        </tr>
+        <?php
+        foreach ($_SESSION["allTasks"] as $tasks) {
```

Comment:
use template php syntax for **foreach, if, while**, etc 
and instead of calling <?php echo ... ?> you can simply use html tags inside
```
<?php foreach ($result as $index => $value): ?>
<span><?= 'gg wp` ?></span>
<?php endforeach; ?>

<?php if (...): ?>
<td><?= 'some text' ?></td>
...
<?php endif; ?>

<?php while (): ?>
<h1><?= 'gg wp' ?></h1>
<?php endwhile; ?>

```